kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -1227,155 +767,33 @@ These rules reach into the network state and load/store from account storage:
     syntax UnStackOp ::= "SLOAD"
  // ----------------------------
     rule [sload]:
-         <k> SLOAD INDEX => #lookup(STORAGE, INDEX) ~> #push ... </k>
-         <id> ACCT </id>
-         <account>
-           <acctID> ACCT </acctID>
-           <storage> STORAGE </storage>
-           ...
-         </account>
+         <k> SLOAD INDEX => GetAccountStorage(INDEX) ~> #push ... </k>
 
     syntax BinStackOp ::= "SSTORE"
  // ------------------------------
     rule [sstore]:
-         <k> SSTORE INDEX NEW => .K ... </k>
-         <id> ACCT </id>
-         <account>
-           <acctID> ACCT </acctID>
-           <storage> STORAGE => STORAGE [ INDEX <- NEW ] </storage>
-           ...
-         </account>
+         <k> SSTORE INDEX NEW => SetAccountStorage(INDEX, NEW) ... </k>
       [preserves-definedness]
 
     syntax UnStackOp ::= "TLOAD"
  // ----------------------------
     rule [tload]:
-         <k> TLOAD INDEX => #lookup(TSTORAGE, INDEX) ~> #push ... </k>
-         <id> ACCT </id>
-         <account>
-           <acctID> ACCT </acctID>
-           <transientStorage> TSTORAGE </transientStorage>
-           ...
-         </account>
+         <k> TLOAD INDEX => GetAccountTransientStorage(INDEX) ~> #push ... </k>
 
     syntax BinStackOp ::= "TSTORE"
  // ------------------------------
     rule [tstore]:
-         <k> TSTORE INDEX NEW => .K ... </k>
-         <id> ACCT </id>
-         <account>
-           <acctID> ACCT </acctID>
-           <transientStorage> TSTORAGE => TSTORAGE [ INDEX <- NEW ] </transientStorage>
-           ...
-         </account>
+         <k> TSTORE INDEX NEW => SetAccountTransientStorage(INDEX, NEW) ... </k>
       [preserves-definedness]
 ```
 
 ### Call Operations
 
 The various `CALL*` (and other inter-contract control flow) operations will be desugared into these `InternalOp`s.
 
--   `#checkCall` takes the calling account and the value of the call and triggers several checks before executing the call.
--   `#checkBalanceUnderflow` takes the calling account and the value of the call and checks if the call value is greater than the account balance.
--   `#checkDepthExceeded` checks if the current call depth is greater than or equal to `1024`.
--   `#checkNonceExceeded` takes the calling account and checks if the nonce is less than `maxUInt64` (`18446744073709551615`).
--   `#call` takes the calling account, the account to execute as, the account whose code should execute, the gas limit, the amount to transfer, the arguments, and the static flag.
--   `#callWithCode` takes the calling account, the account to execute as, the code to execute (as a `Bytes`), the gas limit, the amount to transfer, the arguments, and the static flag.
```

Comment:
We can assume that all these operations are also happening under the PR hood, right?